client/src/app/core/core-services/action.service.ts

```diff
@@ -6,16 +6,36 @@ import { HttpService } from './http.service';
  * Currently supported actions
  */
 export enum ActionType {
+    CREATE = 'create',
```

Comment:
Can this be an own enum? CREATE, UPDATE, DELETE are something like "base action types" in contrast to the concrete actions below.